lib/shouldi/matchers/plug.ex

```diff
@@ -68,7 +70,15 @@ defmodule ShouldI.Matchers.Plug do
   """
   defmatcher should_match_body_to(expected) do
     quote do
-      assert var!(context).connection.resp_body =~ ~r"#{unquote(expected)}"
+      plug_should_match_body_to(unquote(expected), var!(context))
```

Comment:
`=~` accepts both strings and regexes so I would just remove the regex sigil: `assert var!(context).connection.resp_body =~ unquote(expected)`.